tasks/main.yml

```diff
@@ -12,10 +12,10 @@
           incompatible. If trying to build a specific git refspec, use
           "besu_git_refspec" instead of besu_version.
       when: besu_version is defined and besu_build_from_source
-    - name: Check orion_version is not set when fast sync is enabled
+    - name: Check besu_privacy_enabled is not set when fast sync is enabled
       fail:
         msg: Orion and Fast-Sync are incompatible
-      when: orion_version is defined and besu_sync_mode == "FAST"
+      when: ( besu_privacy_enabled|bool == True ) and ( besu_sync_mode == "FAST" )
```

Comment:
Do we expect that this will be version-dependent at some point? I suppose in the interim it's still best to fail fast rather than waiting for the node to complain that it has been misconfigured?